github/gists.go

```diff
@@ -52,6 +52,30 @@ func (g GistFile) String() string {
 	return Stringify(g)
 }
 
+// GistCommit represents a commit on a gist.
+type GistCommit struct {
```

Comment:
I noticed that other structs don't always map those one-to-one according to the doc (maybe I'm missing something though), for example: the `Gist` struct doesn't a URL field (nor `ForksURL` or `CommitsURL`) but they are mentioned in [the docs](https://developer.github.com/v3/gists/#get-a-single-gist). 

I have added the `URL` field to the `GistCommit` and `GistFork` structs.